corehq/apps/hqwebapp/static/hqwebapp/js/hquery.js

```diff
@@ -0,0 +1,12 @@
+// bundles jquery plugins with jquery in one dependency
+// otherwise we'd need to specify which ones are required each time
+// which is nice and explicit, but makes it harder to adapt existing code
+define([
+    "jquery",
+    "jquery.form",
+    "bootstrap",
```

Comment:
Yup, you access bootstrap's JS functions via `$("#id").fn`, so it needs to hook in to jquery.